utils/acquire.go

```diff
@@ -32,10 +32,32 @@ import (
 	"k8s.io/apimachinery/pkg/util/yaml"
 )
 
+const (
+	AnnotationProvenanceFile = "kubecfg.dev/provenance-file"
+	AnnotationProvenancePath = "kubecfg.dev/provenance-path"
```

Comment:
Should we talk about this namespace?  I think these are different to what we use elsewhere in kubecfg.  Should we be consistent, then (maybe) switch them all, or have different values with different prefixes depending on when the feature was added?  (I could be convinced either way)

Reply:
yeah I'd like to switch the old ones to this. I just bought the domain so we have something that's affiliated only with this project

Reply:
an alternative domain name would be `kubecfg.github.com`, in case we don't want to fiddle with domain names (and possibly simplify future maintainership transferals); I quite like a simple `kubecfg.dev` home for the project fwiw.

Reply:
@anguslees how should we proceed?

Reply:
kubecfg.github.com is my favourite for what

Reply:
ok, let's go with `kubecfg.github.com`